alerts/cluster-network-operator/NoRunningOvnControlPlane.md

```diff
@@ -0,0 +1,89 @@
+# NoRunningOvnControlPlane
+
+## Meaning
+
+This alert is triggered when there are no OVN-Kubernetes control plane pods
+[Running][PodRunning] for `10m`.
+
+## Impact
+
+When OVN-Kubernetes control plane is not running, networking control plane is
+degraded and networking configuration updates applied to the cluster will not be
+implemented. Existing workloads may continue to have connectivity but any
+additional workloads may not be provisioned successfully. Network policy changes
```

Comment:
This is outdated now due to IC.
what about:
```
When OVN-Kubernetes control plane is not running, networking control plane is
degraded. A subset of networking control plane functionality is degraded. This includes, but not limited to the following networking control plane functionality:
* Node management including node subnet allocation/de-allocation
* DNS name resolver functionality for EgressFirewall
* EgressService node allocation/re-allocation
* Secondary networks IPAM
* EgressIP assignment/re-assignment and health checks
* OVN zone management
```

Reply:
I was being vague on purpose but I will try a version of what you are proposing